src/test/kotlin/com/kafkakotlin/demo/kafka/usertopology/UserTopologyTest.kt

```diff
@@ -0,0 +1,57 @@
+package com.kafkakotlin.demo.kafka.usertopology
+
+import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
+import com.kafkakotlin.demo.users.User
+import com.kafkakotlin.demo.users.UserSerde
+import org.apache.kafka.common.serialization.Serdes
+import org.apache.kafka.streams.StreamsBuilder
+import org.apache.kafka.streams.StreamsConfig
+import org.apache.kafka.streams.TestInputTopic
+import org.apache.kafka.streams.TopologyTestDriver
+import org.apache.kafka.streams.state.ReadOnlyKeyValueStore
+import org.junit.jupiter.api.AfterEach
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+
+internal class UserTopologyTest() {
+    private lateinit var testDriver: TopologyTestDriver
+    private lateinit var userTopology: UserTopology
```

Comment:
Do you need all these lateinit var class properties? Looks like some are only ever used in the `@BeforeEach`

Reply:
What does that mean, that `private lateinit var` shouldn't be use for things using `@BeforeEach`?

Reply:
No, as in if you don't need something to be at class level e.g. `UserTopology`, `StreamsBuilder` `objectMapper` etc, just keep it in the `BeforeEach`